pkg/controllers/clusterset/syncclusterrolebinding/syncclusterrolebinding_controller.go

```diff
@@ -50,10 +49,6 @@ func NewReconciler(kubeClient kubernetes.Interface,
 
 // Run a routine to sync the clusterrolebinding periodically.
 func (r *Reconciler) Run(ctx context.Context, period time.Duration) {
-	if ok := kubecache.WaitForCacheSync(ctx.Done(), r.clusterRolebindingInformer.Informer().HasSynced); !ok {
-		klog.Errorf("failed to wait for kubernetes caches to sync")
-		return
-	}
 	go utilwait.Forever(r.reconcile, period)
```

Comment:
the outside caller is invoking this func as `go Run()`, so I think the `go` keyword is not necessary, BTW, I think utilwait.UntilWithContext is better than utilwait.Forever

Reply:
and please check others `Run` from other packages.